homeassistant/components/camera/__init__.py

```diff
@@ -399,7 +399,7 @@ async def handle_async_still_stream(
 
     async def handle_async_mjpeg_stream(
         self, request: web.Request
-    ) -> web.StreamResponse:
+    ) -> web.StreamResponse | None:
```

Comment:
If this can return `None` then `handle_async_still_stream` can return `None`.
Then `async_get_still_stream` can return `None`. But it always `web.StreamResponse()`. So how it that possible then?